schemast/annotation.go

```diff
@@ -132,14 +134,18 @@ func protoEnum(annot schema.Annotation) (ast.Expr, bool, error) {
 		},
 	}
 	for k, v := range m.Options {
-		opts.Elts = append(opts.Elts, &ast.KeyValueExpr{
-			Key:   strLit(k),
-			Value: intLit(int(v)),
-		})
-	}
-	sort.Slice(opts.Elts, func(i, j int) bool {
-		return opts.Elts[i].(*ast.KeyValueExpr).Value.(*ast.BasicLit).Value < opts.Elts[j].(*ast.KeyValueExpr).Value.(*ast.BasicLit).Value
-	})
+		opts.Elts = append(
```

Comment:
Thanks for all the formatting suggestions in this PR, but as they are not core to this change (and are a matter of style), can we discuss them in a separate PR?

Reply:
yea sure, just default intellij settings for me :) will revert these changes.